### ISSUE TITLE
Getting TokenMismatchException after laravel session lifetime even after 0.3.11 caffeine version.

### ISSUE THREAD

**Issue by lfsaestrella** [GeneaLabs/laravel-caffeine#56]
# Issue
Hello. I'm getting TokenMismatchException after Laravel session lifetime, just like closed issue #22. I did the same test as ivanvermeyen did: I set the session lifetime to 3 minutes and caffeine drips to 1 minute. I've checked the chrome network tab and the drips are being sent just like they should be. After 3 minutes (or 3 drips) I submit the form and then I get the TokenMismatchException. 
I also noticed one thing: When I did the same test as ivanvermeyen, I had to publish the config file and customize it. After that I'm getting this error when I run an artisan command: 

**PHP Catchable fatal error:  Argument 2 passed to Illuminate\Routing\UrlGenerator::__construct() must be an instance of Illuminate\Http\Request, null given**

When I delete the config file, I don't get this error anymore.

I don't know if it has something to do with the TokenMismatchException error, but I think it's a relevant issue.

## My enviroment:
Laravel Version: *5.2.45*
Laravel Caffeine Package Version: *0.3.11*
PHP Version: *5.6.22*

**Comment by mikebronner** [comment]
Give version 0.3.12 a try, let me know if the error still persists.

**Comment by lfsaestrella** [comment]
Same issue, it still throws the exception. The same goes on the artisan command when I publish the config file.

**Comment by mikebronner** [comment]
oh! yes, I remember ... you have to remove the `url('\')` helper command from the config file. That was fixed only in version 0.4 I believe. Replace it with `null` instead, so your config file should look like this:

```
return [
    'dripIntervalInMilliSeconds' => 300000,
    'domain' => null,
    'route' => 'genealabs/laravel-caffeine/drip',
];
```

**Comment by lfsaestrella** [comment]
I replaced the `url('\')` and this is what happened: The network tab from chrome returned a 404 error from the drip, instead of the usual 204. The replacement fixed the artisan problem, but now it can't find the drip route. Also, the original issue had not been solved, as the TokenMismatchException continues to be thrown.
